tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -325,6 +325,10 @@ def __init__(self, test_ctx: TestContext, *args, **kwargs):
         if self.redpanda._cloud_cluster.config.provider == PROVIDER_AWS:
             self._agent_services.append('redpanda-agent-init.service')
 
+    def tearDown(self):
+        super().tearDown()
+        self.redpanda.clean_cluster()
```

Comment:
@savex i'm expecting it to be safe to all delete topics on test teardown because the cloudv2 tests are run serially. correct me if i'm wrong on this assumption.

Reply:
That's correct. Moreover, we might want to do the same in setup with a check and warning as there might be rare situations when ducktape:run_client thread will terminate unexpectedly without teardown.

Reply:
i added `clean_cluster()` to `startup()` as well